static/routes.config.php

```diff
@@ -188,6 +188,10 @@
 		'/{sub1}/{sub2}/{url}' => [Module\Proxy::class, [R::GET]],
 	],
 
+	'/search' => [
+		'/acl'                 => [Module\Search\Acl::class, [R::GET, R::POST]],
```

Comment:
It seems like there's only a post route available, isn't it?

Reply:
No, we use `$_REQUEST` in `rawContent`, so it is available with any method, but we only *use* POST at the moment in the Javascript.

Reply:
ok